Examples/ObjectiveC/CalloutDelegateUsageExample.m

```diff
@@ -69,13 +69,16 @@ - (void)mapView:(MGLMapView *)mapView annotation:(id<MGLAnnotation>)annotation c
 {
     // Hide the callout view.
     [self.mapView deselectAnnotation:annotation animated:NO];
+    
+    // Create an alert containing the annotation's details
+    UIAlertController* alert = [UIAlertController alertControllerWithTitle:annotation.title
+                                                                   message:@"A lovely (if touristy) place."
+                                                            preferredStyle:UIAlertControllerStyleAlert];
+    UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"Ok" style:UIAlertActionStyleDefault
```

Comment:
As you’ve done in the Swift example, how about combining the action definition (this) and `addAction:` into one call.